test/service/test_request_validation.py

```diff
@@ -284,10 +296,36 @@ def test_bad_filter_relation(self):
         }
         relation_message = {
             'Code': 'BadRequestError',
-            "Message": "BadRequestError: The relation in the `filters` parameter entry for `organPart` must be one of"
-                       " ['is', 'contains', 'within', 'intersects']"
+            'Message': "BadRequestError: The relation in the 'filters' parameter entry for 'organPart' must be one of"
+                       " '['is', 'contains', 'within', 'intersects']'."
         }
         response = requests.get(url, params=params)
         print(response.json())
         self.assertEqual(400, response.status_code, response.json())
         self.assertEqual(relation_message, response.json())
+
+    def test_bad_nested_relation_value(self):
+        path = '/index/files'
+        url = furl(url=self.base_url, path=path)
+        facet = 'organPart'
+        for relation in ['contains', 'within', 'intersects']:
+            path_specs = _extract_parameters(api_spec=self.app_module.app.specs,
+                                             resource_path=path,
+                                             http_method='get')
+            filter_spec = next(x for x in path_specs if x['name'] == 'filters')
+            spec_location = ['content', 'application/json', 'schema', 'properties', facet, 'oneOf']
+            relation_spec = next(x for x in reduce(getitem, spec_location, filter_spec) if relation in x['properties'])
```

Comment:
Using `more_itertools.one` would make this stronger.
```suggestion
            filter_spec = one(x for x in path_specs if x['name'] == 'filters')
            spec_location = ['content', 'application/json', 'schema', 'properties', facet, 'oneOf']
            relation_spec = one(x for x in reduce(getitem, spec_location, filter_spec) if relation in x['properties'])
```